package.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+set -e
+set -u
+
+VERSION=`/source/usr/local/bin/fortio version | cut -f 1 -d ' '`
+echo $VERSION
+# how that we have our version, let's build DEB
+cd /source
+fpm -s dir -t deb -n fortio -v $VERSION .
+cp fortio*.deb /packages
```

Comment:
how does that get outside of the image ?

Reply:
you mount packages outside of the container, in the "host"

Reply:
is there an example / end to end of how that works, or a script for that part ? (see for instance how I get the tgz out of the image for the other binary build)